src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12153,6 +12153,53 @@ public void TestAnalyzerFilteringBasedOnSeverity(DiagnosticSeverity defaultSever
                     output, StringComparison.Ordinal);
             }
         }
+
+        [DiagnosticAnalyzer(LanguageNames.CSharp)]
+        private sealed class FieldAnalyzer : DiagnosticAnalyzer
+        {
+            private static readonly DiagnosticDescriptor _rule = new DiagnosticDescriptor("Id", "Title", "Message", "Category", DiagnosticSeverity.Warning, isEnabledByDefault: true);
+
+            public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(_rule);
+
+            public override void Initialize(AnalysisContext context)
+            {
+                context.RegisterSyntaxNodeAction(AnalyzeFieldDeclaration, SyntaxKind.FieldDeclaration);
+            }
+
+            private static void AnalyzeFieldDeclaration(SyntaxNodeAnalysisContext context)
+            {
+            }
+        }
+
+        [Fact]
+        [WorkItem(44000, "https://github.com/dotnet/roslyn/issues/44000")]
+        public void TupleField_ForceComplete()
```

Comment:
>public void TupleField_ForceComplete() [](start = 8, length = 38)

Consider adding direct test for the symbol. I.e. calling ForceComplete on the tight field symbol.